presidio-anonymizer/presidio_anonymizer/entities/engine/result/decrypt_result_item.py

```diff
@@ -0,0 +1,44 @@
+"""Result item for the /decrypt method."""
```

Comment:
You should also change the name of the file to match the class name

Reply:
Done.